### PR TITLE
Add user id param to account creation

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/models.py [hunk 1]
```diff
@@ -205,6 +206,10 @@ def _get_valid_user_name(cls, user_name):
     def _get_email(cls, user):
         return user.preferences.get('wl_email', user.email)
 
+    @classmethod
+    def _generate_userid(cls, user):
```
Comment (RayeN) [inline sitewit/models.py]: `_generate_user_id`?

File: sitewit/models.py [hunk 2]
```diff
@@ -205,6 +206,10 @@ def _get_valid_user_name(cls, user_name):
     def _get_email(cls, user):
         return user.preferences.get('wl_email', user.email)
 
+    @classmethod
```
Comment (RayeN) [inline sitewit/models.py]: it might be `@staticmethod` as well

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/services.py [hunk 3]
```diff
@@ -68,14 +68,15 @@ def _get_partner_auth_headers(
...
-    def create_account(self, url, user_name, user_email,
+    def create_account(self, url, user_id, user_name, user_email,
                        currency, country_code, site_id=None,
                        mobile_phone=None, user_token=None,
                        remote_subpartner_id=None):
         """Create new SiteWit account.
 
         Args:
             url: (str): site URL.
+            user_id (str): id of account owner.
```
Comment (RayeN) [inline sitewit/services.py]: We should probably mention here, that it's expected to be in `email` format

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/models.py [hunk 4]
```diff
@@ -205,6 +206,10 @@ def _get_valid_user_name(cls, user_name):
...
+    def _generate_user_id(cls, user):
```
Comment (RayeN) [inline sitewit/models.py]: Just thinking... What if we call it `_generate_sw_user_id` just to distinguish it from Yola's `user_id`?

Comment (ekoval) [reply sitewit/models.py]: makes sense

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**